internal/version/version.go

```diff
@@ -25,11 +25,9 @@ import (
 var (
 	// version is the current version of Helm.
 	// Update this whenever making a new release.
-	// The version is of the format Major.Minor.Patch[-Prerelease][+BuildMetadata]
```

Comment:
This version does apply once a tag is built. This statement holds true.

https://github.com/helm/helm/blob/a2d11c92883509f283b9edd84a203b6e49a9580a/Makefile#L38-L41

Can you please reinstate this?

Reply:
```
 ><> helm version
version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"clean", GoVersion:"go1.13.12"}
```